customer-account-extension/locales/fr.json

```diff
@@ -1,3 +1,3 @@
 {
-  "welcome": "Bienvenue dans l'extension {{target}} de l'interface utilisateur du compte client!"
+  "welcome": "Bienvenue dans la {{target}} de l'extension d'interface utilisateur du compte client!"
```

Comment:
Following from above, did this go through the translation team?

```suggestion
  "welcome": "Bienvenue à la cible {{target}} de l'extension d'interface utilisateur du compte client!"
```

Reply:
>  did this go through the translation team?

No, I just use goolge translate to do it. 😄

Reply:
Updated.